Gaijinizer patch/Map196.txt

```diff
@@ -145,28 +153,29 @@ Was this a goal?#MTLed
 \\C[2]『心蝕』、『キズナ』、そしてあの『悪魔全書』……
 つまるところ、悪魔との関わりと、まぐわいじゃ
 > CONTEXT: Map196/events/1/pages/0/147/Dialogue/enemy17
-What an "intervener" shares with \\N[1] ...
-\\C[2]"Mind corruption", "Kizuna", and that "all
-demons" ...... After all, the relationship with
-the demon#MTLed
+The things the "intervener" shared with \\N[1].
+\\C[2]"Corruption", "Bonds", and that "Complete Book
+of Demons"... In short, the connections with
+demons, as well as sexual intercourse...#CheckLater
 > END STRING
```

Comment:
Minor cleanup:

>_**The things the "Intervener" shared with \\\N[1]..
"Bonds", "Corruption", and that "Compendium"...
Additionally, all the connections that he made with
those demons...**_